server/utils.go

```diff
@@ -173,3 +176,129 @@ func fixGithubNotificationSubjectURL(url string) string {
 func fullNameFromOwnerAndRepo(owner, repo string) string {
 	return fmt.Sprintf("%s/%s", owner, repo)
 }
+
+// filter lines in a string from start to end
+func filterLines(s string, start, end, limit int) string {
+	scanner := bufio.NewScanner(strings.NewReader(s))
+	var buf strings.Builder
+	i := 1
+	w := 1
+	for scanner.Scan() {
```

Comment:
can you merge `i` into this for loop since we don't need `w` to track the lines

Reply:
This is now gone because of the simplification.